plugins/woocommerce/src/Admin/RemoteInboxNotifications/NotRuleProcessor.php

```diff
@@ -11,6 +11,14 @@
  * Rule processor that negates the rules in the rule's operand.
  */
 class NotRuleProcessor implements RuleProcessorInterface {
+
+	/**
+	 * The rule evaluator to use.
+	 *
+	 * @var RuleEvaluator
+	 */
+	protected $rule_evaluator;
```

Comment:
Since this property was being dynamically created before, it was assumed "public". I opted to use "protected" to declare it since I couldn't find a single instance in our codebase accessing the property from the outside and it does look like it should be internal.